resources/js/views/Statistics.vue

```diff
@@ -18,10 +18,10 @@
 	</Panel>
 	<Activity v-if="!is_se_preview_enabled" />
 	<Panel class="max-w-5xl mx-auto border-0" :pt:header:class="'hidden'">
-		<TotalCard v-if="total" :total="total" />
-		<div class="py-4" v-if="load && total">
-			<ToggleSwitch v-model="is_collapsed" class="text-sm"></ToggleSwitch> {{ $t("statistics.collapse") }}
-		</div>
+		<template v-if="load && total !== undefined && showTotal">
```

Comment:
` && total !== undefined` is also checked in `showTotal`

Reply:
yes, but unfortunately TypeScript is not smart enough to make this inferance.
```ts
const showTotal = computed(() => isTotalValue(total.value));
function isTotalValue(total: TotalAlbum | undefined): total is TotalAlbum {
	return total !== undefined && (total.num_albums > 0 || total.num_photos > 0 || total.size > 0);
}
```
here is what happens if I remove it:
![image](https://github.com/user-attachments/assets/97335b01-ef06-4f38-a24f-f0a8f37bfc7c)

Reply:
:man_facepalming: